config/locales/doc_auth/es.yml

```diff
@@ -184,16 +196,21 @@ es:
       email_sent: Enlace enviado a %{email}. Compruebe el correo electrónico de su
         escritorio y siga las instrucciones para verificar su identidad.
       learn_more: Aprende más
+      privacy: Login.gov es un sitio web gubernamental seguro que se adhiere a los
+        más altos estándares en protección de datos. Solo usamos sus datos para verificar
+        su identidad. Obtenga más información sobre nuestras medidas de privacidad
+        y seguridad.
```

Comment:
This should be the title text? It currently duplicates title and body content.

![image](https://user-images.githubusercontent.com/1779930/112151511-ae087d00-8bb7-11eb-8b30-62cd7246e30f.png)

Reply:
@stevegsa title text here should be: 

> Nuestras normas de privacidad y seguridad

Reply:
Also, the Learn more link in the translation I got for this is: **Obtenga más información**, but here it says: **Aprende más**.  Is there a "Learn more" component with its own translation that we are using throughout?  (Not a Spanish speaker so not sure which is more correct)